src/static/js/modules/jquery/cf_notifier.js

```diff
@@ -14,33 +14,32 @@ var _notifierTemplate = '<div class="cf-notification cf-notification__{{ state }
 
 var _notifier = {
   defaults: {
-    message:    'There was an error with your submission',
-    state:      'error',
-    speed:      400,
-    easing:     'swing',
-    template:   _notifierTemplate,
-    input:      null,
-    onRender:   null,
-    onClearAll: null,
-    onDestroy:  null
+    message:        'There was an error with your submission',
+    state:          'error',
+    speed:          400,
+    easing:         'swing',
+    template:       _notifierTemplate,
+    insertLocation: 'prependTo',
+    insertTarget:   null,
+    input:          null,
+    onRender:       null,
+    onClearAll:     null,
+    onDestroy:      null
   },
 
   // Generate a string of HTML from the plugin's settings
   // @returns {string} The expanded HTML string
   _generateHTML: function() {
+    var icon = {
+      error:   'delete',
+      success: 'approved',
+      warning: 'error',
+    };
     var data = {
       message: _notifier.settings.message,
-      state:   _notifier.settings.state
+      state:   _notifier.settings.state,
+      icon:    icon[_notifier.settings.state]
     };
```

Comment:
I wasn't sure if it was worth redefining it inside the function, but since it's used so many times I figured you are right. Updated in 3c08f0f